packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2336,6 +2352,17 @@ class SemanticsConfiguration {
     _hasBeenAnnotated = true;
   }
 
+  /// The name of a widget subtree.
```

Comment:
Update this doc comment since route is a bool and not a name?

Reply:
done